core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -157,4 +158,42 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener, conf: Spar
     }
   }
 
+  private def updateExecutorBlacklist(eid: String, isBlacklisted: Boolean): Unit = {
+    val execTaskSummary = executorToTaskSummary.getOrElseUpdate(eid, ExecutorTaskSummary(eid))
+    execTaskSummary.isBlacklisted = isBlacklisted
+  }
+
+  override def onExecutorBlacklisted(executorBlacklisted: SparkListenerExecutorBlacklisted)
+  : Unit = synchronized {
```

Comment:
nit: move param to second line, and double-indent.  (and the other added methods in this file)

```scala
override def onExecutorBlacklisted(
    executorBlacklisted: SparkListenerExecutorBlacklisted): Unit = synchronized {
```